app/components/keywords/edit_component.html.erb

```diff
@@ -1 +1,11 @@
-<%= render Elements::Forms::TextFieldComponent.new(form:, field_name: :text, label: helpers.t('keyword.edit.fields.text.label')) %>
```

Comment:
Moved away from the text field because there's no seam currently to omit the `mb-4` class on the containing `<div>`, which creates a gap between the keyword input field and the dropdown with FAST values.

Reply:
Can we do a separate ticket / PR to (1) move `mb-4` outside the component and (2) make this a `TextFieldComponent`?

I think we discussed this previously and decided the `mb-4` was probably a bad idea.

Reply:
@justinlittman Good call: #323